<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomGameParamWindow</class>
 <widget class="QDialog" name="CustomGameParamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomGameParamWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="Canada"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Height :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="heightSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="widthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>78</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="nbMinesRadioButton">
          <property name="text">
           <string>Number of mines :</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">methodButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="nbMinesSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="densityRadioButton">
          <property name="text">
           <string>Density of mines :</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">methodButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="densitySpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="difficultyRadioButton">
          <property name="text">
           <string>Difficulty :</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">methodButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="difficultyComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLength</enum>
          </property>
          <item>
           <property name="text">
            <string>Super Easy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Easy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Super Hard</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomGameParamWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomGameParamWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="methodButtonGroup"/>
 </buttongroups>
</ui>
